src/components/Recipe.js

```diff
@@ -0,0 +1,93 @@
+import React, { useEffect, useState } from "react";
+import { Link, useParams, useHistory } from "react-router-dom";
+import {
+  Title,
+  Button,
+  CircleList,
+  CircleListItem,
+  Container,
+} from "./Styles.js";
+import api from "../data/DataAPI.js";
+
+function IngredientList(props) {
+  const { ingredients } = props;
+  return (
+    <CircleList>
+      {ingredients &&
+        ingredients.map((ingredient) => (
+          <CircleListItem key={ingredient.name}>
+            {ingredient.name}
+          </CircleListItem>
+        ))}
+    </CircleList>
+  );
+}
+
+function RecipeDetail({ recipe }) {
+  const history = useHistory();
+
+  const deleteRecipe = () => {
+    let url = `${process.env.REACT_APP_API_BASEURL}/api/recipe/recipes/${recipe.id}/`;
+
+    fetch(url, { method: "DELETE" }).then(() => history.push("/recipes/"));
+  };
+
+  return (
+    <div>
+      {recipe ? (
+        <div>
+          <Title data-testid="recipe_title">{recipe.name}</Title>
```

Comment:
you don't need a `data-testid` here to check some text is on the page just use

`expect(screen.getByText('my-text')).toBeInTheDocument()`